lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -701,7 +716,7 @@ html.video-fullscreen {
           overflow: hidden;
         }
       }
-    }
+    }z
```

Comment:
Typo?

Reply:
oops,
 fixed thanks @ormsbee for looking for it.